packages/examples/src/Feedback/Feedback.tsx

```diff
@@ -1,4 +1,4 @@
-import { graphql, useFragment, useMutation } from "@nova/react";
+import { graphql, useFragment, useMutation_deprecated } from "@nova/react";
```

Comment:
Let's not use deprecated functions in examples - they should be as up to date as possible

Reply:
Would that mean we need to update graphitation first? If so I would be in favor of that as otherwise nova bump in apollo based hosts will require search and replace and TBH I am not even sure how would we handle that in people app

Reply:
Yes, that would mean that we need to update graphitation first. We discussed it with Mark, and the initial plan was to start with this change. But we can revisit the initial decision